docs/core/node.md

```diff
@@ -1,10 +1,11 @@
 <!--
 order: 4
-synopsis: The main endpoint of an SDK application is the daemon client, otherwise known as the full-node client. The full-node runs the state-machine, starting from a genesis file. It connects to peers running the same client in order to receive and relay transactions, block proposals and signatures. The full-node is constituted of the application, defined with the Cosmos SDK, and of a consensus engine connected to the application via the ABCI. 
 -->
 
 # Node Client (Daemon)
 
+The main endpoint of an SDK application is the daemon client, otherwise known as the full-node client. The full-node runs the state-machine, starting from a genesis file. It connects to peers running the same client in order to receive and relay transactions, block proposals and signatures. The full-node is constituted of the application, defined with the Cosmos SDK, and of a consensus engine connected to the application via the ABCI. {synopsis}
```

Comment:
How do you spot the synopsis without frontmatter?

Reply:
We used to have synopsis in front-matter, but we've switched to `{synopsis}` syntax.